src/electionguard/decryption/trustee.c

```diff
@@ -124,21 +125,21 @@ Decryption_Trustee_new(uint32_t num_trustees, uint32_t threshold,
     return result;
 }
 
-void Decryption_Trustee_free(Decryption_Trustee d)
+void Decryption_Trustee_free(Decryption_Trustee decryption_trustee)
```

Comment:
all changes to this file are only to improve readability and fix spelling errors.  There are no logic changes